package.json

```diff
@@ -113,8 +113,6 @@
   },
   "version": "4.0.4",
   "dependencies": {
-    "curriable": "^1.1.0",
-    "fast-stringify": "^1.1.1",
```

Comment:
This is removed in favor of a custom `replacer` to `JSON.stringify`, and `json-prune` hasn't been used for quite a while.